Challenge-4/Bot_1/src/agent.ts

```diff
@@ -15,60 +16,87 @@ import {
   SUPPLY_KINK,
   BORROW_KINK,
   SUPPLY_PYIR,
+  USDC_TOKEN_ETH,
+  USDC_TOKEN_ARB,
 } from "./constants";
 import { createFinding } from "./findings";
-import { getCollateralAsset } from "./helper";
-import { data, getAddress, getConfigurator } from "./networkManager";
+import { getCollateralAsset, getAddress, getConfigurator } from "./helper";
+import { NetworkManager } from "forta-agent-tools";
+
+let configuratorProxy: string | undefined;
+let networkManager: any;
+interface NetworkData {
+  usdc: string;
+  configurationproxy: number;
+}
+const data: Record<number, NetworkData> = {
+  1: {
+    usdc: USDC_TOKEN_ETH,
+    configurationproxy: 1,
+  },
+  42161: {
+    usdc: USDC_TOKEN_ARB,
+    configurationproxy: 2,
+  },
```

Comment:
error stating `Type 'string' is not assignable to type 'number'.ts(2322)
agent.ts(30, 3): The expected type comes from property 'configurationproxy' which is declared here on type 'NetworkData'`, if I change it

Reply:
Yes, so you'd need to update the `NetworkData` interface you declared so that `configurationproxy` is of type `string` instead of `configurationproxy: number;`. This the strong typing of TypeScript coming into play. Luckily, since _you_ declared it, you can just change what it is that it's supposed to expect. 😉

Reply:
Actually I was going to do that, but you reviewed it before I could implement it